tables/subject_plasma/gagne22-plasma.csv

```diff
@@ -0,0 +1,25 @@
+﻿ref_name,subject_name,rx_name,collection_date_cmp,collection_date,location,cumulative_group,section
+Gagne22,mRNA1273_3dose,mRNA1273_2dose_wk6,~,2021-03-15,USA,mRNA1273_3dose,NULL
+Gagne22,mRNA1273_3dose,mRNA1273_2dose_wk6,~,2021-03-15,USA,mRNA1273_3dose,NULL
+Gagne22,mRNA1273_3dose,mRNA1273_2dose_wk6,~,2021-03-15,USA,mRNA1273_3dose,NULL
+Gagne22,mRNA1273_3dose,mRNA1273_2dose_wk6,~,2021-03-15,USA,mRNA1273_3dose,NULL
+Gagne22,mRNA1273_3dose,mRNA1273_2dose_wk41,~,2021-11-15,USA,mRNA1273_3dose,NULL
+Gagne22,mRNA1273_3dose,mRNA1273_2dose_wk41,~,2021-11-15,USA,mRNA1273_3dose,NULL
+Gagne22,mRNA1273_3dose,mRNA1273_2dose_wk41,~,2021-11-15,USA,mRNA1273_3dose,NULL
+Gagne22,mRNA1273_3dose,mRNA1273_2dose_wk41,~,2021-11-15,USA,mRNA1273_3dose,NULL
+Gagne22,mRNA1273_3dose,mRNA1273_3dose_wk43,~,2021-11-29,USA,mRNA1273_3dose,NULL
+Gagne22,mRNA1273_3dose,mRNA1273_3dose_wk43,~,2021-11-29,USA,mRNA1273_3dose,NULL
+Gagne22,mRNA1273_3dose,mRNA1273_3dose_wk43,~,2021-11-29,USA,mRNA1273_3dose,NULL
+Gagne22,mRNA1273_3dose,mRNA1273_3dose_wk43,~,2021-11-29,USA,mRNA1273_3dose,NULL
+Gagne22,mRNA1273_2dose_mRNAOmicron_boost,mRNA1273_2dose_wk6,~,2021-03-15,USA,mRNA1273_2dose_mRNAOmicron_boost,NULL

```

Comment:
I think the CI failed, because two different **subject name**s are pointing to the same **rx_name**(mRNA1273_2doese_wk6)

Reply:
Yes, in this case we have two subject sets: one that takes an mRNA-1273 boost, and another that takes an mRNA-1273.529 boost. Both groups take the same two-dose mRNA-1273 regimen. The issue I'm facing is that both groups are treated as one for the week 6 and week 41 isolations (they are in the same aggregate potency value) but then are split for the week 43 isolation.

Reply:
Would I be able to split the week 6 and week 41 observations into separate mRNA-1273 and mRNA-1273.529 groups? In other words, turn the 8-count observations into two 4-observation counts, with each sharing the same potency values?

Reply:
Seems to me a bit complex situation. Let me check the paper.

Reply:
@zachandfox how about using this method.

For week 6 and 41, use a new vaccine name called "mRNA-1273 or mRNA-1273.529" both from Moderna. But for week 43, use separated vaccine "mRNA-1273", "mRNA-1273.529".

In the vaccines table. I use **+** to indicate the two vaccines are used in one subject. and use **or** to indicate the different subjects are using the different vaccines, like "BNT162b2 or mRNA-1273".

Reply:
Except in week 6 and 41 all subjects use "mRNA-1273", and in week 43 it is either one of "mRNA-1273" or "mRNA-1273.529"

Binning them all into one subject "mRNA1273_3dose" did not work

Reply:
@zachandfox, for week 43, please use different subject names. The method is to distinguish subjects in week 43. Because we can't distinguish subjects in weeks 6 and 41 we use the same subjects. This means we will have three different subject names, two for week 43 and one for both week 6 and 41.

Another way to process this special paper is to manually calculate the median of week 43 and treat it as the titer of week 43 for both vaccinee groups.